tests/plugin/text/oos_filter/test_oos_filter.py

```diff
@@ -0,0 +1,32 @@
+from dialogy.base import Input, Output
+from dialogy.plugins.text.oos_filter import OOSFilterPlugin
+from dialogy.types.intent import Intent
+from dialogy.workflow import Workflow
+
+
+def test_oos_filter() -> None:
```

Comment:
Whenever you write unit test, each unit test should be as atomic as possible. Here you are checking two different cases. This is how you can rewrite the test to maintain atomicity:

```
pytest.mark.parameterize("intent_name, predicted_confidence, threshold, expected_intent", [("already_paid", 0.05, 0.5, "_oos_"), ... <more test cases here>]
def test_oos_filter(intent_name, predicted_confidence, threshold, expected_intent) -> None:
    intent_oos = "_oos_"
    oos_filter = OOSFilterPlugin(
        dest="output.intents",
        threshold=threshold,
        replace_output=True,
        intent_oos=intent_oos,
    )

    # Create a mock workflow
    workflow = Workflow([oos_filter])
    body = [[{"transcript": "I have already paid"}]]
    intent = Intent(name=predicted_intent, score=predicted_confidence)

    output = Output(intents=[intent])
    _, output = workflow.run(Input(utterances=body), output)
    assert output.intents[0].name == expected_intent
```